src/main/java/servlets/calculator.java

```diff
@@ -0,0 +1,158 @@
+package servlets;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.*;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.annotation.WebServlet;
+
+
+@WebServlet(
+        name = "calculator",
+        urlPatterns = {"/calc"}
+)
+
+public class calculator extends HttpServlet{
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The class name 'calculator' doesn't match '[A-Z][a-zA-Z0-9]*'](https://app.codacy.com/manual/thejerome/IFMO_Servlets_20190915/pullRequest?prid=4854440)